xml/System.Text.Json/JsonElement.xml

```diff
@@ -741,10 +752,24 @@ For other types, the value of <xref:System.Text.Json.JsonElement.GetRawText> is
         <Parameter Name="value" Type="System.Byte[]" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">When the method returns, contains the byte representation of <paramref name="value" /> if the method succeeded.</param>
```

Comment:
byte[] or byte array

```suggestion
        <param name="value">When the method returns, contains the byte[] representation of <paramref name="value" /> if the method succeeded.</param>
```

Reply:
I think in this case, using natural language sounds a bit better: byte array. I'll modify your suggestion and commit it.